azerothcore/azerothcore-wotlk
# fix(scripts/ZulGurub): Update Gahzranka's abilities

Pull Request by [sílker](https://www.chromiecraft.com/en/armory/?character/ChromieCraft/s%C3%ADlker)



## Changes Proposed:
- Disable triggered spells
- Adjust init spell events
- Adjust CD timers
- Add thrash spell/event

## Issues Addressed:

- Closes #11562
- Closes #11568

## SOURCE:


https://www.youtube.com/watch?v=JEmFTdr65UY
The encounter starts at 1:00

## Tests Performed:

- None

## How to Test the Changes:


1. Enter ZUl'Gurub
2. Engage the boss
3. Check spell casting


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
